conformance/utils/suite/suite.go

```diff
@@ -470,6 +471,13 @@ func (suite *ConformanceTestSuite) Run(t *testing.T, tests []ConformanceTest) er
 		if res == testSucceeded || res == testFailed {
 			sleepForTestIsolation = true
 		}
+
+		// call the hook function if it was provided,
+		// this's useful for running custom logic after each test,
+		// such as collecting current state of the cluster for debugging.
+		if suite.Hook != nil {
```

Comment:
nit: is it only useful on the suite level, meaning it will run the same hook for each test? or there is value in doing it on a more granular level like tests/profile/supportedFeature etc

Reply:
yes, it's for suite level.
For specified test, you can use `t.Cleanup()`